docs/general/configuration_options.md

```diff
@@ -9,6 +9,8 @@ preferably inside an initializer.
 
 - `adapter`: The [adapter](adapters.md) to use. Possible values: `:attributes, :json, :json_api`. Default: `:attributes`.
 - `serializer_lookup_enabled`: When `false`, serializers must be explicitly specified. Default: `true`
+- `default_includes`: What relationships to serialize by default.  Default: `'*'`, which includes one level of related
+  objects. See [includes](adapters.md#included) for more info.
```

Comment:
Hmm. I wrote this to match the config key, although I can see it being a little disconcerting to click through and find a different header. Of course, the thing actually being described is the `include` option, which doesn't match either of these 100%. And, of course, it's a little weird to be linking to docs for the JSON API adapter to show behavior that is also applicable to adapters like `Attributes`. :disappointed: